frontend/src/pages/projects/screens/spawner/SpawnerPage.tsx

```diff
@@ -144,9 +144,11 @@ const SpawnerPage: React.FC<SpawnerPageProps> = ({ existingNotebook }) => {
               <NameDescriptionField
                 nameFieldId="workbench-name"
                 descriptionFieldId="workbench-description"
+                nameHelperText="Must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character"
                 data={nameDesc}
                 setData={setNameDesc}
                 autoFocusName
+                isNameValid
```

Comment:
This should either provide the validated value or provide a validation function.
```suggestion
                isNameValid={isValidNotebookName(nameDesc.name)}
```